lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -143,12 +155,32 @@ for barclamp in $deactivation_list; do
 done
 
 
+### Stop chef-clients at the nodes, to ensure services shutdown in next step won't be started by chef
+for node in $allocated_nodes; do
+  echo "Stopping chef-client service on ${node}..."
+  ssh "$node" 'rcchef-client stop'
+done
+
+
+### Stop pacemaker resources in all clusters and put the nodes into standby mode
+for founder in $cluster_founders; do
+  echo_summary "Stopping crm resources on ${founder}..."
```

Comment:
This should be echo, not echo_summary. And I think the block should be after the `echo_summary "Stopping OpenStack services..."` bit.